jep/1/README.adoc

```diff
@@ -0,0 +1,939 @@
+ifdef::env-github[]
+:tip-caption: :bulb:
+:note-caption: :information_source:
+:important-caption: :heavy_exclamation_mark:
+:caution-caption: :fire:
+:warning-caption: :warning:
+endif::[]
+
+= JEP-1: Jenkins Enhancement Proposal Format
+
+:toc:
+
+.Metadata
+[cols="2"]
+|===
+| JEP
+| 1
+
+| Title
+| Jenkins Enhancement Proposal Format
+
+| Sponsor
+| link:https://github.com/rtyler[R Tyler Croy]
+
+| Status
+| :speech_balloon: Draft
+
+| Type
+| Process
+
+| Created
+| 2017-09-12
+
+| Discussion-To
+| link:https://groups.google.com/d/msg/jenkinsci-dev/spDAr8EJm3c/T9Nmhn-fAQAJ[jenkinsci-dev@]
+
+|===
+
+
+[[abstract]]
+== Abstract
+
+A Jenkins Enhancement Proposal (JEP) is a design document that
+describes a new feature or aspect of Jenkins itself,
+or the Jenkins Project processes or environment.
```

Comment:
Capitalization inconsistent with https://jenkins.io/press/#branding (I prefer this one, but still).